firmware/controllers/algo/launch_control.cpp

```diff
@@ -198,20 +199,25 @@ void setDefaultLaunchParameters(DECLARE_CONFIG_PARAMETER_SIGNATURE) {
 	engineConfiguration->enableLaunchBoost = true;
 	engineConfiguration->launchSmoothRetard = true; //interpolates the advance linear from launchrpm to fully retarded at launchtimingrpmrange
 	engineConfiguration->antiLagRpmTreshold = 3000;
+
 }
 
 void applyLaunchControlLimiting(bool *limitedSpark, bool *limitedFuel DECLARE_ENGINE_PARAMETER_SUFFIX) {
-
-	if (retardThresholdRpm < GET_RPM()) {
+	if (( engine->isLaunchCondition ) && ( retardThresholdRpm < GET_RPM() )) {
 		*limitedSpark = engineConfiguration->launchSparkCutEnable;
 		*limitedFuel = engineConfiguration->launchFuelCutEnable;
+	} else {
+		*limitedSpark = false;
+		*limitedFuel = false;
 	}
 }
 
 void initLaunchControl(Logging *sharedLogger DECLARE_ENGINE_PARAMETER_SUFFIX) {
 	logger = sharedLogger;
-	retardThresholdRpm = RETART_THD_CALC;
-	Launch.Start();
+	retardThresholdRpm = CONFIG(launchRpm) +\
```

Comment:
sloppy copy paste `\`?

Reply:
do we have a max length accepted for a line? i wanted to keep shorter so it can be visible on the screen

Reply:
you only need the `\` when in a macro - normal code can span multiple lines